### ISSUE TITLE
Bug: android crashes if level has no hints

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#373]
```
                 Caused by: java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
                                                                                       at rabbitescape.ui.android.Dialogs$IntroDialogs.hint(Dialogs.java:95)
                                                                                       at rabbitescape.ui.android.Dialogs$IntroDialogs.show(Dialogs.java:63)
                                                                                       at rabbitescape.ui.android.Dialogs.intro(Dialogs.java:168)
                                                                                       at rabbitescape.ui.android.AndroidGameActivity.onCreate(AndroidGameActivity.java:97)
                                                                                       at android.app.Activity.performCreate(Activity.java:6251)
```

Tests should insist on hints, or android code should be more gracious in the absence of hints.

**Comment by andybalaam** [comment]
I think levels that are in development should definitely be playable without hints. We may also want to demand hints in official levels.

**Comment by colonelfazackerley** [comment]
fixed in  7a52fdb84320257da6c8e967c1c44937e34d08a2
